Clients/GUI/Scripts/SignUpper.cs

```diff
@@ -0,0 +1,18 @@
+using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+using HttpWebRequestWrapperLib;
+
+namespace GUI.Scripts
+{
+    public class SignUpper
+    {
+        public static void SignUp(SignUpViewModel input)
```

Comment:
We already have `public static void SignIn(UserEmailPassword userInput)`. It's better to keep one type of arguments

Reply:
If this method, for example, takes n seconds, then other users will not be able to get a page with a log up about n seconds

Reply:
Fixed